### DISCUSSION TITLE
do workspaces support `server` config option?

### DISCUSSION THREAD

**Discussion by husayt** [vitest-dev/vitest#4377] [Q&A]
I added this line to my project's vitest config
```
    server: {
      deps: {
        inline: [/my-library/],
      },
    },
```

When I run vitest for locally for that project it all works fine, but when I run same test via workspace this config gets ignored.

The question is if server config not supported with workspaces? Or maybe this a vitest issue?

**Comment by sheremet-va** [comment]
They should support it, yes
